authalligator_client/exceptions.py

```diff
@@ -41,8 +39,7 @@ class ExecutionResultError(AuthAlligatorException):
 class AccountError(ExecutionResultError):
     """Corresponds with the AccountError entity type."""
 
-    def __init__(self, code, message, retry_in, *args):
-        # type: (enums.AccountErrorCode, Optional[Union[str, bytes]], Optional[int], *Any) -> None
+    def __init__(self, code: enums.AccountErrorCode, message: Optional[Union[str, bytes]], retry_in: Optional[int], *args):
```

Comment:
I wonder if the message can still be `bytes` in Py3.

Reply:
This is indeed created from JSON response of GraphQL mutation, so it will be always str on Python 3.